examples/example-gateway/build/endpoints/bar/workflow/bar_bar_method_argnotstruct.go

```diff
@@ -74,8 +74,12 @@ func (w barArgNotStructWorkflow) Handle(
 	if ok {
 		clientHeaders["X-Zanzibar-Use-Staging"] = h
 	}
+	h, ok = reqHeaders.Get("x-trace-id")
```

Comment:
the casing is not consistent with existing header casing in this file.

Reply:
although under the hood we camelcase the header passed in, but good to stay case consistency

Reply:
I can update the codegen to format/canonicalize whatever headers are in gateway.json, instead of preserving the original casing. Although right now the library is case insensitive with `Get` and canonicalizes all headers that are set